parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -215,18 +214,21 @@ notifyUser dir o = case o of
     -- do
     -- Console.clearScreen
     -- Console.setCursorPosition 0 0
-  Typechecked sourceName errorEnv uf -> do
-    Console.setTitle "Unison ☺︎"
-    -- todo: we should just print this the same way as everything else
-    let defs       = prettyTypecheckedFile uf errorEnv
-    if (not $ null defs) then putPrettyLn' . ("\n" <>) . P.okCallout . P.lines $
-     [ P.wrap $ "I found and" <> P.bold "typechecked" <> "these definitions in " <> P.group (P.text sourceName <> ":")
-     , ""
-     , P.lit defs
-     , P.wrap "Now evaluating any watch expressions (lines starting with `>`)..."
-     ]
+  Typechecked sourceName ppe uf -> do
+    Console.setTitle "Unison ✅"
+    let terms = sortOn fst [ (HQ.fromVar v, typ) | (v, _, typ) <- join $ UF.topLevelComponents uf ]
+        typeDecls =
+          [ (HQ.fromVar v, Left e)  | (v, (_,e)) <- Map.toList (UF.effectDeclarations' uf) ] ++
+          [ (HQ.fromVar v, Right d) | (v, (_,d)) <- Map.toList (UF.dataDeclarations' uf) ]
+    if UF.nonEmpty uf then putPrettyLn' . ("\n" <>) . P.okCallout . P.sep "\n\n" $ [
+      P.wrap $ "I found and" <> P.bold "typechecked" <> "these definitions in "
+            <> P.group (P.text sourceName <> ":"),
+      P.indentN 2 . P.sepNonEmpty "\n\n" $ [
```

Comment:
This is a good idiom if you have two or more lists of things that you want separated by blank lines.